src/tests/routes/crates/reverse_dependencies.rs

```diff
@@ -10,14 +11,16 @@ async fn reverse_dependencies() {
 
     app.db(|conn| {
         let c1 = CrateBuilder::new("c1", user.id).expect_build(conn);
-        CrateBuilder::new("c2", user.id)
+        let c2 = CrateBuilder::new("c2", user.id)
             .version(VersionBuilder::new("1.0.0").dependency(&c1, None))
             .version(
                 VersionBuilder::new("1.1.0")
                     .dependency(&c1, None)
                     .dependency(&c1, Some("foo")),
             )
             .expect_build(conn);
+        let _ = update_default_version(c1.id, conn);
+        let _ = update_default_version(c2.id, conn);
```

Comment:
might be better to run `update_default_version()` in the `expect_build()` fn instead?

Reply:
I've been thinking about this as well. However, I'm concerned that it might become more difficult to simulate scenarios where default_versions doesn't catch up (even though we don't have such a scenario currently).

Do you think it would be more suitable to have an option to toggle with_default_versions or another build_expect that controls syncing default_versions? Or would that be too complicated?

Reply:
I think if we want to test such a scenario we can manually adjust the value in the `default_versions` table as needed, but by default it should update it correctly.

Reply:
Make sense.